### ISSUE TITLE
NTR: SARS-CoV-2 coronavirus RT-PCR in sputum or pharyngeal sample positive (Lab: bacteriology, virology, parasitology, fungal)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#7916]
New term request SARS-CoV-2 coronavirus RT-PCR in sputum or pharyngeal sample positive (nadea3_200415142906): Reverse transcriptase polymerase chain reaction; COVID-19; RNA  UMLS: 5244037

**Comment by pnrobinson** [comment]
@MickeySegal
Adding new term: 
- ID: HP:6000837
- label: Positive respiratory tract SARS-CoV-2 coronavirus nucleic acid test
- parents: [Positive respiratory tract infectious agent test (HP:0430151)](https://hpo.jax.org/app/browse/term/HP:0430151)
- pmid: 
- def: Detection of SARS-CoV-2 coronavirus nucleic acid in the sputum or pharyngeal sample by a method such as polymerase chain reaction.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/7916
- synonyms:
